jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/definition/list/ProcessDefinitionListPresenter.java

```diff
@@ -159,7 +158,7 @@ public boolean error(Message message,
                                                return onRuntimeDataServiceError();
                                            }
                                        }).getProcessesByFilter(getSelectedServerTemplate(),
-                                                               textSearchStr,
+                                                               null,
```

Comment:
Just one question here.
Instead of introducing ```null``` argument, why not switching to ```getProcesses``` method?